tests/test_pay.py

```diff
@@ -1748,7 +1746,9 @@ def listpays_nofail(b11):
 def test_pay_routeboost(node_factory, bitcoind, compat):
     """Make sure we can use routeboost information. """
     # l1->l2->l3--private-->l4
-    l1, l2 = node_factory.line_graph(2, announce_channels=True, wait_for_announce=True)
+    # Note: l1 gets upset because it extracts update for private channel.
```

Comment:
I'm not sure I understand the comment here. Since we trigger "Bad gossip" only when receiving messages from peers, does this mean that we are accidentally sending a private update for which we don't have a matching announcement (being private)?

Reply:
Hmm, good question!  I've removed the suppression of bad gossip detection, and am re-running.

Reply:
Yes, we are.  Apparently Eclair will actually use these, though we don't currently.

Should we suppress them?